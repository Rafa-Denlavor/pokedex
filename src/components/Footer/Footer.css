.footer {
  background-color: var(--color-primary);
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: space-around;
}

.logo {
  width: 190px;
  padding: 20px 0;
}

.top {
  border: 2px solid var(--color-white);
  padding: var(--gap-8) var(--gap-8) 0 var(--gap-8);
  border-radius: var(--gap-8);
  scroll-behavior: smooth;
}

.top:target {
  scroll-margin-top: .8em;
}

.top img {
  width: 20px;
}
