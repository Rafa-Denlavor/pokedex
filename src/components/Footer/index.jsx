import './Footer.css';
import React from 'react';

const Footer = () => {
    return (
        <div className='footer'>
            <img className='logo' src='images/logo.svg' alt="Logo Pokemon" />
        </div>
    )
}

export default Footer;