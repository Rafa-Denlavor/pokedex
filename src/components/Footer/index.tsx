import './Footer.css';
import React from 'react';

interface Footer {
    logo?: string;
    path: string;
    arrow?: string;
}

const Footer = ({logo = 'images/logo.svg', path, arrow = 'images/seta.svg'} : Footer) => {

    return (
        <div className='footer'>
            <img className='logo' src={logo} alt="Logo Pokemon" />
            <a className='top' href={path}>
                <img src={arrow} alt='Seta para cima' />
            </a>
        </div>
    )
}

export default Footer;