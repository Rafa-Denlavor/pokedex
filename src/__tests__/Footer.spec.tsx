import React from "react";
import { render } from "@testing-library/react";
import Footer from "../components/Footer";

describe("Footer", () => {
  test("renders learn react link", () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
