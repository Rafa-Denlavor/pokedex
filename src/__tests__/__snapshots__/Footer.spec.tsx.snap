// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders learn react link 1`] = `
<div>
  <div
    class="footer"
  >
    <img
      alt="Logo Pokemon"
      class="logo"
      src="images/logo.svg"
    />
  </div>
</div>
`;
